web/src/lib/components/shared-components/search-bar/search-filter-modal.svelte

```diff
@@ -168,6 +173,9 @@
         <!-- DISPLAY OPTIONS -->
         <SearchDisplaySection bind:filters={filter.display} />
       </div>
+
+      <!-- RATING -->
+      <SearchRatingsSection bind:rating={filter.rating} />
```

Comment:
This option should only show up when the star rating feature is enabled

Reply:
I'll go ahead and make the change, but kind of going back and forth on whether it makes the most sense. Since ratings can be set outside of Immich before uploading, it could still be valuable to be able to filter by star rating even if the star rating feature is disabled. At the same time I could see it being confusing to have this option visible here but not see a way to set ratings on the asset if you have the filter always visible even if star ratings are disabled

I could go either way tbh so happy to hear ppl's thoughts

Reply:
I agree with Alex and I think disabling the "star rating" feature should disable it everywhere, including in the search :)